### PR TITLE
adding 2023-07-11 data

### PR DESCRIPTION
**Pull request by tracykteal**

### CODE REVIEW

**Comment by tracykteal** [comment]
Closes https://github.com/rfordatascience/tidytuesday/issues/462

Thanks @BB1464

File: data/2023/2023-07-11/post_vars.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "article_link": "https://data.giss.nasa.gov/gistemp/",
+  "pic1_alt": "Line plot of global mean temperature per month. Peak temperatures are in July and lowest temperatures are in December and January. Global average temperatures have increased approximately 2 degrees since the 1880s. Derived from the MERRA2 reanalysis over 1980-2015. We show how much warmer each month of the GISTEMP data is than the annual global mean. Graph data: The 1980-2015 seasonal cycle anomaly in MERRA2 along with the 95% uncertainties on the estimate of the mean.",
+  "pic2_alt": "Global map showing temparture differences in 2023 compared with the time period 1951-1980. Most areas of the map indicate that temperatures are 1 to 2 degrees warmer. Areas of northern Canada, northern Russia and Antarctica are up to 6.5 degees warmer. Some areas over oceans are the same or slightly cooler. An an estimate for the global mean of the calculated field is 0.94 degrees."
```
Comment (jonthegeek) [inline data/2023/2023-07-11/post_vars.json:4]:
```suggestion
  "pic2_alt": "Global map showing temperature differences in 2023 compared with the time period 1951-1980. Most areas of the map indicate that temperatures are 1 to 2 degrees warmer. Areas of northern Canada, northern Russia and Antarctica are up to 6.5 degees warmer. Some areas over oceans are the same or slightly cooler. An estimate for the global mean of the calculated field is 0.94 degrees."
```

File: data/2023/2023-07-11/readme.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+tweets](https://docs.ropensci.org/rtweet/reference/post_tweet.html) with
+alt text programatically.
+
+Need a **reminder**? There are
+[extensions](https://chrome.google.com/webstore/detail/twitter-required-alt-text/fpjlpckbikddocimpfcgaldjghimjiik/related)
+that force you to remember to add Alt Text to Tweets with media.
+
+# Global surface temperatures 
+
+The data this week comes from the [NASA GISS Surface Temperature Analysis (GISTEMP v4)](https://data.giss.nasa.gov/gistemp/). This dataset are tables of global and hemispheric monthly means and zonal annual means. They combine land-surface, air and sea-surface water temperature anomalies (Land-Ocean Temperature Index, L-OTI). The values in the tables are deviations from the corresponding 1951-1980 means.
```
Comment (jonthegeek) [inline data/2023/2023-07-11/readme.md:50]:
```suggestion
The data this week comes from the [NASA GISS Surface Temperature Analysis (GISTEMP v4)](https://data.giss.nasa.gov/gistemp/). This datasets are tables of global and hemispheric monthly means and zonal annual means. They combine land-surface, air and sea-surface water temperature anomalies (Land-Ocean Temperature Index, L-OTI). The values in the tables are deviations from the corresponding 1951-1980 means.
```

File: data/2023/2023-07-11/readme.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+# Read in with tidytuesdayR package 
+# Install from CRAN via: install.packages("tidytuesdayR")
+# This loads the readme and all the datasets for the week of interest
+
+# Either ISO-8601 date or year/week works!
+
+tuesdata <- tidytuesdayR::tt_load('2023-07-11')
+tuesdata <- tidytuesdayR::tt_load(2023, week = 28)
+
+global_temps <- tuesdata$`global_temps`
```
Comment (jonthegeek) [inline data/2023/2023-07-11/readme.md:76]:
```suggestion
global_temps <- tuesdata$global_temps
```

File: data/2023/2023-07-11/readme.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+nh_temps <- tuesdata$`nh_temps`
```
Comment (jonthegeek) [inline data/2023/2023-07-11/readme.md:77]:
```suggestion
nh_temps <- tuesdata$nh_temps
```

File: data/2023/2023-07-11/readme.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+sh_temps <- tuesdata$`sh_temps`
```
Comment (jonthegeek) [inline data/2023/2023-07-11/readme.md:78]:
```suggestion
sh_temps <- tuesdata$sh_temps
```

File: data/2023/2023-07-11/readme.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+zonann_temps <- tuesdata$`zonann_temps`
```
Comment (jonthegeek) [inline data/2023/2023-07-11/readme.md:79]:
```suggestion
zonann_temps <- tuesdata$zonann_temps
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
A couple typos that I'll commit, then I'll merge the whole thing 🎉

**Pull request merged**